subsys/fs/nvs/Kconfig

```diff
@@ -27,6 +27,23 @@ config NVS_LOOKUP_CACHE_SIZE
 	  Number of entries in Non-volatile Storage lookup cache.
 	  It is recommended that it be a power of 2.
 
+config NVS_LOOKUP_CACHE_ARRAY_SIZE
+	int "Non-volatile Storage lookup cache size"
+	default 128
+	range 1 65536
+	depends on NVS_LOOKUP_CACHE
+	help
+	  Number of entries in Non-volatile Storage lookup cache.
+	  It is recommended that it be a power of 2.
+
+
+config NVS_SAVE_RAM_EN
```

Comment:
config NVS_RAM_OPTIMIZATION
    bool "Enable RAM optimization for NVS lookup cache"
    default n
    depends on NVS_LOOKUP_CACHE
    help
      Enable this option to optimize RAM usage for NVS by using a specialized RAM-based lookup cache.